### PR TITLE
Adding Licence.txt and Readme pointer

### PR DESCRIPTION
**Pull request by mathewc**
Putting the same license + readme pointer that we use for our other repos (e.g. https://github.com/Azure/azure-webjobs-sdk)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,9 +71,13 @@ To run the test script located [here](test/TemplateTest.ps1), Execute the follow
 - blobTrigger
 - timerTrigger
 
+## License
+
+This project is under the benevolent umbrella of the [.NET Foundation](http://www.dotnetfoundation.org/) and is licensed under [the MIT License](https://github.com/Azure/azure-webjobs-sdk-templates/blob/master/LICENSE.txt)
```
Comment (soninaren) [inline README.md]: Nit: Can use a relative link for license.txt. Would make the License link work on dev branch as well.

Comment (mathewc) [inline README.md]: How?

Comment (soninaren) [inline README.md]: change the link to (LICENSE.txt)

Comment (mathewc) [inline README.md]: Good suggestion. Done.

**Comment by soninaren** [comment]
looks good. :shipit:

**Pull request merged**